content/docs/guides/components/istio.md

```diff
@@ -85,6 +99,9 @@ like
 See istio [doc](https://istio.io/docs/tasks/telemetry/metrics-logs.html).
 
 #### Expose Grafana dashboard behind ingress/IAP
+
+TODO(lunkai): update this section
```

Comment:
Can we put TODOs in comments so that won't be rendered? Can we also link TODOs to issues instead of people?

Reply:
link to issue instead of person now.

I think render the TODO is better so that people know this part is not up-to-date.